cdc/sorter/unified/unified_sorter.go

```diff
@@ -170,8 +170,16 @@ func (s *Sorter) Run(ctx context.Context) error {
 		}
 	})
 
+	merger := &sorterMerger{
```

Comment:
Maybe we need a constructor function here?

Reply:
done